tools/termux-installation.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+nav_order: 2
+parent: Applications and Variants
+---
+
+# What is this?
+
+Using this, you can install Termux, a linux "emulator" for Android. The Quest 2 happens to run android, so you can host the SlimeVR server on it using this.
+
+***joycons still require a PC for now and the Wrangler: https://github.com/carl-anders/slimevr-wrangler
+
+# INSTRUCTIONS - QUEST 2
+1. Install a file explorer on your quest, CX File Explorer is good. You can get the APK from https://apkpure.com
```

Comment:
Do you need to? The other instructions don't seem like they would.

Reply:
It made sense for the original way the installer worked, forgot to change it.